utilities/account/fixing_parent/fixingAccountParent.js

```diff
@@ -0,0 +1,96 @@
+// Fixing account parent
+'use strict';
+
+var db = require('../../../server/lib/db'),
+		q  = require('q');
+
+db.initialise();
+
+(function queryAccount() {
+	var accounts;
```

Comment:
The local variable `accounts` is unneeded.  You can keep the accounts data within the promise chain.  Here's how:
```js
executeUpdate()
.then(executeAccount) // executeAccount returns a list of accounts, so I'll pass that list to the next function
.then(unmatchAccount) // unmatchAccount receives a list of accounts from execute account, and returns the same list at the end of execution.
.then(updateParent);  // updateParent receives a list of accounts from unmatchAccount()
```

Now, you just have to change the function signature of unmatchParent to this:
```
function unmatchParent(accounts) {
  /* code */
  return accounts;
}
```
Also, you have to change updateParent's signature.
```js
function updateParent(accounts) {
  /* code */
}
```